openstudiocore/src/model/ModelCore.i

```diff
@@ -168,7 +168,7 @@ namespace model {
     return os.str();
   }
 
-  // This really shouldn't be necessary
```

Comment:
@jmarrec you the grammar police now? ;-)

Reply:
I replaced “shouldn’t” by “should not” because the ‘ was breaking syntax highlighting for the rest of the file on my vim editor :)